db/migrate/20151215154514_alter_visit_change_spelling_of_canceled_to_cancelled.rb

```diff
@@ -0,0 +1,15 @@
+class AlterVisitChangeSpellingOfCanceledToCancelled < ActiveRecord::Migration
+  def up
+    execute(
+      "UPDATE visits SET processing_state = 'cancelled' " +
+      "WHERE processing_state = 'canceled'"
+    )
```

Comment:
You can use `<<-` to avoid unnecessary concatenation (and GitHub will even syntax-highlight it as SQL!):

```ruby
execute <<-SQL
  UPDATE visits SET processing_state = 'cancelled'
  WHERE processing_state = 'canceled'
SQL
```